eq-author-api/middleware/version.js

```diff
@@ -0,0 +1,15 @@
+module.exports = (req, res, next) => {
+  if (
+    process.env.NODE_ENV !== "development" &&
```

Comment:
Yeah the reason for this is because it just causes a bunch of additional logging noise every time you make a request to the API as while in dev the version numbers aren't present, this seemed like a better solution than putting dummy version numbers in the envs?

Reply:
If we change to something like this it won't log in the event that versions aren't set. No need for dummy versions to be entered in dev.

```js
if (req.headers.clientversion && process.env.EQ_AUTHOR_API_VERSION && req.headers.clientversion !== process.env.EQ_AUTHOR_API_VERSION) {
    req.log.warn(
      {
        clientversion: req.headers.clientversion,
        apiversion: process.env.EQ_AUTHOR_API_VERSION,
      },
      "Application version mismatch"
    );
  }
  next();
};
```